packages/control-property-editor/src/panels/outline/Tree.tsx

```diff
@@ -298,7 +298,9 @@ export const Tree = (): ReactElement => {
 
         const tooltipId = `tooltip--${item?.name}`;
 
-        const cellName = hasDefaultContent ? `${item?.name} (has default content)` : item?.name;
+        const cellName = hasDefaultContent
+            ? t('EXTENSION_POINT_HAS_DEFAULT_CONTENT_TEXT', { name: item?.name })
+            : item?.name;
```

Comment:
For some reason, onRenderCell handler has an item as optional. Well, it is because the nestingDepth is optional but we do not use it, then the item should also be optional because: `A required parameter cannot follow an optional parameter.`

```ts
const onRenderCell = (nestingDepth?: number, item?: OutlineNodeItem, itemIndex?: number): React.ReactNode => {
```

I do think that we will always have a name no matter if this item is optional. Just to be safe this check should stay?